modules/op-deleting-pipelines.adoc

```diff
@@ -0,0 +1,8 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+[id="op-deleting-pipelines_{context}"]
+= Deleting Pipelines
+. In the *Pipelines* view of the *Developer* perspective, click the *Options* {Kebab} menu adjoining a pipeline, and select *Delete Pipeline*.
```

Comment:
```suggestion
. In the *Pipelines* view of the *Developer* perspective, click the *Actions* {Kebab} menu adjoining a pipeline, and select *Delete Pipeline*.
```
We label the Kebab menu Actions on the details pages, perhaps use that instead of Options?

Reply:
Maybe don't do this - you use `Options` everywhere. I'll leave it around for you to think on it... but I personally think consistency with the rest of the docs makes sense over this different naming.

Reply:
I will leave this as Options. OCP docs refer to Kebab menu as options menu as a convention, and the curly brackets will render to show the 3 vertical dots. Also Actions menu is on the details page, and not on this one. I have added that point in the last step of the Constructing module.